CDCgov/prime-reportstream
# 11662: MSG to Coding, PT to Meta mappings

This PR adds the mappings for MSG to Coding and PT to Meta

Test Steps:
1. Verify the tests pass
2. Verify the mappings match the inventory spreadsheets

## Changes
- MSG to Coding mapping
- PT to Meta mapping

## Checklist

### Testing

### Process

## Linked Issues
- #11662 

## To Be Done

## Specific Security-related subjects a reviewer should pay specific attention to
